server/@types/dto/index.d.ts

```diff
@@ -54,6 +54,34 @@ declare module 'dto' {
     howIdentifiedOther?: string
   }
 
+  export interface ChallengeResponseDto {
```

Comment:
I was initially wondering if we could somehow re-use the existing `ChallengeDto` perhaps by adding fields to it, or extending the type in some way ....
But in the end I think you are right, it is a different class in it's own right 👍